2022-01-12 09:41:57,642 INFO    MainThread:4781 [wandb_setup.py:_flush():69] setting env: {}
2022-01-12 09:41:57,642 INFO    MainThread:4781 [wandb_setup.py:_flush():69] setting login settings: {}
2022-01-12 09:41:57,642 INFO    MainThread:4781 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220112_094157-n06znhsm/logs/debug.log
2022-01-12 09:41:57,643 INFO    MainThread:4781 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220112_094157-n06znhsm/logs/debug-internal.log
2022-01-12 09:41:57,643 INFO    MainThread:4781 [wandb_init.py:init():369] calling init triggers
2022-01-12 09:41:57,643 INFO    MainThread:4781 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-01-12 09:41:57,644 INFO    MainThread:4781 [wandb_init.py:init():418] starting backend
2022-01-12 09:41:57,644 INFO    MainThread:4781 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-12 09:41:57,681 INFO    MainThread:4781 [backend.py:ensure_launched():123] starting backend process...
2022-01-12 09:41:57,687 INFO    MainThread:4781 [backend.py:ensure_launched():128] started backend process with pid: 4831
2022-01-12 09:41:57,688 INFO    MainThread:4781 [wandb_init.py:init():423] backend started and connected
2022-01-12 09:41:57,688 INFO    MainThread:4781 [wandb_init.py:init():465] updated telemetry
2022-01-12 09:41:57,689 INFO    MainThread:4781 [wandb_init.py:init():484] communicating current version
2022-01-12 09:41:58,445 INFO    MainThread:4781 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-12 09:41:58,445 INFO    MainThread:4781 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-01-12 09:41:58,993 INFO    MainThread:4781 [wandb_init.py:init():522] starting run threads in backend
2022-01-12 09:42:02,523 INFO    MainThread:4781 [wandb_run.py:_console_start():1524] atexit reg
2022-01-12 09:42:02,524 INFO    MainThread:4781 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-01-12 09:42:02,525 INFO    MainThread:4781 [wandb_run.py:_redirect():1376] Redirecting console.
2022-01-12 09:42:02,525 INFO    MainThread:4781 [wandb_run.py:_redirect():1438] Redirects installed.
2022-01-12 09:42:02,525 INFO    MainThread:4781 [wandb_init.py:init():546] run started, returning control to user process
2022-01-12 09:42:09,139 INFO    MainThread:4781 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-01-12 09:42:09,140 INFO    MainThread:4781 [wandb_run.py:_restore():1466] restore
2022-01-12 09:42:11,623 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 837
}

2022-01-12 09:42:12,014 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 1204
}

2022-01-12 09:42:12,759 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:12,862 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:12,967 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:13,071 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:13,174 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:13,278 INFO    MainThread:4781 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6888
  total_bytes: 6888
}

2022-01-12 09:42:14,575 INFO    MainThread:4781 [wandb_run.py:_show_files():1838] logging synced files
